CONTRIBUTORS.md

```diff
@@ -10,6 +10,8 @@
 -                     Ryan D Ripken (ryan@rmanet.com)
 - Copyright 2020-2024 Resource Management Associates
 -                     Adam N. Korynta (adam@rmanet.com)
+- Copyright 2020-2024 Resource Management Associates
```

Comment:
I know the other ones have a range but I'm pretty sure that should be just 2024.

Reply:
Fixed that.